CHANGELOG.md

```diff
@@ -13,6 +27,8 @@
 
 ### Fixed
 
+- `DatePickerInput`: moved inputProps on top of `onClick` and `onFocus`, to make sure the component handlers will be called ([@eniskraasniqi1](https://github.com/eniskraasniqi1) in [#2285](https://github.com/teamleadercrm/ui/pull/2285)).
+
 ### Dependency updates
 
 - [BREAKING] `node`: set minimum `node` version to `14.15.0` to bump `postcss-loader` ([@farazatarodi](https://github.com/farazatarodi) in [#2288](https://github.com/teamleadercrm/ui/pull/2288)).
```

Comment:
looks like it needs to be `16.0.0`

Reply:
I will close this one and create a new one for 16.0.0

Reply:
I just replied to the 16.0.0 PR asking why it would be a major update. The fix for the inputProps is actually a bugfix and is not a breaking change to me.